components/items/ItemsGrid/ItemsGridImageTokenEntity.vue

```diff
@@ -70,7 +70,7 @@
           />
         </template>
 
-        <template v-else-if="listVisible(urlPrefix)">
```

Comment:
why?

Reply:
item transfer is available for all VMs 

`listVisible` is used inside `ListingCartMini.vue`

![CleanShot 2024-10-21 at 15 56 18@2x](https://github.com/user-attachments/assets/349b3f9c-6100-45dc-b00a-dc251e26de7a)